cmake/ConfigGen.cmake

```diff
@@ -60,7 +60,7 @@ function(generate_export_configs)
   endif()
 
   configure_file("cmake/Templates/GreenteaLibDNNConfig.cmake.in"
-           "${PROJECT_BINARY_DIR}/GreenteaLibDNNConfig.cmake" @ONLY)
+           "${PROJECT_BINARY_DIR}/GreenteaConfig.cmake" @ONLY)
 
```

Comment:
first we generate this: https://github.com/edgarriba/libdnn/blob/8c6ef64aa81da55c9f27bbad5eb3d3682051f31c/cmake/ConfigGen.cmake#L62

then this: https://github.com/edgarriba/libdnn/blob/8c6ef64aa81da55c9f27bbad5eb3d3682051f31c/cmake/ConfigGen.cmake#L86-L87